pages/docs/tutorials/getting-started/slackbot-websocket-tutorial.md

```diff
@@ -0,0 +1,340 @@
+---
```

Comment:
Hi Vaisihnavi, 
Your tutorial is placed in the wrong section i.e. `https://www.asyncapi.com/docs/tutorials/getting-started`, 
![image](https://github.com/asyncapi/website/assets/52382282/548a8f63-110e-4ae5-96c8-d68db396f94b)

You'll need to move it into the parent directory i.e. `https://www.asyncapi.com/docs/tutorials`

Reply:
Thanks @J0SAL !!